domainobjs/ts/__tests__/publicKey.test.ts

```diff
@@ -1,49 +1,192 @@
 import { expect } from "chai";
-import { unpackPubKey } from "maci-crypto";
+import { SNARK_FIELD_SIZE, unpackPubKey } from "maci-crypto";
 
 import { Keypair, PubKey } from "..";
 
 describe("public key", () => {
-  it("isValidSerializedPubKey() should work correctly", () => {
-    const k = new Keypair();
-    const s = k.pubKey.serialize();
+  describe("constructor", () => {
+    it("should create a public key", () => {
+      const k = new Keypair();
+      const pk = k.pubKey;
+      expect(pk).to.not.eq(null);
+    });
+    it("should create the same public key object for the same raw key", () => {
+      const k = new Keypair();
+      const pk1 = k.pubKey;
+      const pk2 = k.pubKey;
+      expect(pk1.rawPubKey.toString()).to.eq(pk2.rawPubKey.toString());
```

Comment:
Not quite sure what's going on here. Thinking about generate `Keypair()` from same given `PrivKey` and check they should be equal?

Reply:
Oh no idea either, totally messed up there my bad 😆 will fix it properly